server/controllers/finance/reports/invoicedReceivedStock/index.js

```diff
@@ -0,0 +1,203 @@
+const _ = require('lodash');
+const db = require('../../../../lib/db');
+const ReportManager = require('../../../../lib/ReportManager');
+const Patients = require('../../../medical/patients');
+
+const TEMPLATE = './server/controllers/finance/reports/invoicedReceivedStock/report.handlebars';
+// expose to the API
+exports.report = report;
+
+// default report parameters
+const DEFAULT_PARAMS = {
+  csvKey : 'invoiced_received_stock',
+  filename : 'REPORT.PATIENT_STANDING.DESCRIPTION',
```

Comment:
This filename should reflect what the report is.  The `filename` property is used to name the file when exporting to PDF.